client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -199,6 +188,37 @@ class CalypsoifyIframe extends Component<
 		window.removeEventListener( 'message', this.onMessage, false );
 	}
 
+	/**
+	 * The editor redirect timer will attempt to perform a redirect if the editor
+	 * has not loaded after a certain time. There are currently two stages to the.
```

Comment:
Nit: the full stop mark was probably intended to be after `timer` in the next line?

Reply:
thanks!